corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -39,7 +39,6 @@
 {% include 'hqwebapp/includes/ui_element_js.html' %}
 {% compress js %}
   <script src="{% static 'markdown-it/dist/markdown-it.js' %}"></script>
-  <script src="{% static 'jquery-tiny-pubsub/dist/ba-tiny-pubsub.js' %}"></script>
```

Comment:
So this dependency was unused? Would it makes sense to remove it from yarn.lock/package.json as well?

Reply:
It's used - see the reference in `form_ui.js`. The script tag isn't needed anymore because of requirejs. I tried to remove script tags from this file as I migrated the code relying on them (this file acted as something of a TODO list), but I got sloppy and have some commits like https://github.com/dimagi/commcare-hq/pull/34271/commits/258ff20dea980cfb0e60761f920b1f7c001a4063 that just remove a bunch of script tags.